Echo.Process.Redis/RedisClusterImpl.cs

```diff
@@ -302,9 +302,31 @@ public IEnumerable<T> GetQueue<T>(string key)
             Retry(() =>
                 Db.HashSet(
                     key, 
-                    fields.Map((k, v) => new HashEntry(k, JsonConvert.SerializeObject(v))).ToArray()
+                    fields.Map((_, v) => new HashEntry(v.Key, JsonConvert.SerializeObject(v.Value))).ToArray()
```

Comment:
Just a separate bug: k here was coming in as int (index) rather than key 
`Map((int,(string,T)) => ...)` instead of `Map((string, T) =>..)`